valuesets/ValueSet-UKCore-HumanLanguage.xml

```diff
@@ -20,979 +20,31 @@
       <rank value="1" />
     </telecom>
   </contact>
-  <description value="A set of codes that define a language used by a person." />
+  <description value="A set of codes that define a language used by a person.Selected from the SNOMED CT UK coding system:
```

Comment:
Do we want to revert this valueset to its original contents if we are retiring it?